src/decaylanguage/utils/utilities.py

```diff
@@ -53,7 +53,7 @@ def filter_lines(matcher, inp):
     Filter out lines into new variable if they match a regular expression
     """
     output = [
-        matcher.match(ln).groupdict() for ln in input if matcher.match(ln) is not None
+        matcher.match(ln).groupdict() for ln in inp if matcher.match(ln) is not None
     ]
-    new_inp = [ln for ln in input if matcher.match(ln) is None]
+    new_inp = [ln for ln in inp if matcher.match(ln) is None]
```

Comment:
This is a bug, probably introduced by me recently.

Reply:
Could have nasty effects! Nice catch.